topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md

```diff
@@ -0,0 +1,842 @@
+
+title: Genome-wide alternative splicing analysis
+zenodo_link: ''
+questions:
+- Are there statistically significant different genome splicing patterns between healthy and tumor samples?
+- Are there ssignificant isoform switching events between healthy and tumor samples?
+objectives:
+- Perform genome-wide isoform analyis in order to evaluate diffences in isoform expression profiles between cancer and healthy tissues
+- Evaluate the differential splicing in specific genes
+time_estimation: 3H
+key_points:
+- Before start analyzing RNA-seq data, it is important to evaluate the quality of the samples by using RSeQC.
+- StingTie can be used for gene isoform quantification
+- SwitchIsoformAnalyzer can be used for evaluate the statistical siggnificance of isoform switching events in different conditions
+
+contributors:
+- gallardoalba
+---
+
+# Introduction
+
+Discovered over 40 years ago, alternative splicing formed a large part of the puzzle explaining how proteomic complexity can be achieved with a limited set of genes  ({% cite Alt1980 %}). The majority of eukaryote genes have multiple transcriptional isoforms, and recent data indicate that each transcript of protein-coding genes contain 11 exons and produce 5.4 mRNAs on average ({% cite Piovesan2016 %}). In humans,  approximately 95% of multi-exon genes show evidence of alternative splicing (AS) and approximately 60% of genes have at least one alternative transcription start site, some of which exert antagonistic functions ({% cite Carninci2006 %}, {% cite Miura2012 %}).
+
+AS regulation is essential for providing cells and tissues their specific features, and for their response to environmental changes ({% cite Wang2008 %}, {% cite Kalsotra2011 %}). Differential usage of isoforms in different conditions, often referred to as isoform switching (IS), can have substantial biological impact, caused by the difference in the functional potential of the two isoforms ({% cite VittingSeerup2017 %}).
+
+Extensive alternative splicing programmes contribute to the regulation of cell differentiation and organ development, and its disruption has been demostranted to be able contribute to virtually every aspect of cancer progression ({% cite Namba2021 %}, {% cite Bonnal2020 %}) (fig. 1). Cancer is a disease characterized by the uncontrolled growth and spread of abnormal cells in the body. It can start almost anywhere in the human body, which is made up of trillions of cells. From a molecular point of view, cancer is caused by changes to genes that control cell function, particularly how they grow and divide. There are more than 100 types of cancer, and they are usually named for the organs or tissues where they form. Some common types of cancer include carcinomas, sarcomas, leukemias, and lymphomas.
+
+![figX:Isoform usage](../../images/differential_isoform/differential_cancer.png "Effect of alternative splicing dysregulation on cancer progression. The diagram depicts various cancer hallmarks.")
```

Comment:
the alt text for this image maybe isn't great. Could you instead provide a brief summary of the figure's conclusions? (i.e. if this was an audiobook, what would be useful to hear about this figure?) the figure caption (effect..) is great as is though!

Reply:
Yes, it was a important missing part, thanks :D